src/recipes.jl

```diff
@@ -550,7 +550,7 @@ Plots.@deps stepbins path
 function _auto_binning_nbins{N}(vs::NTuple{N,AbstractVector}, dim::Integer; mode::Symbol = :auto)
     _cl(x) = max(ceil(Int, x), 1)
```

Comment:
Hm, maybe we better do

    _cl(x) = ceil(Int, max(x, one(x)))

instead, just in case. Well spotted!